src/NServiceBus.MessagingBridge/Configuration/BridgeConfiguration.cs

```diff
@@ -54,7 +54,12 @@ internal FinalizedBridgeConfiguration FinalizeConfiguration(ILogger<BridgeConfig
             if (tranportsWithNoEndpoints.Any())
             {
                 var endpointNames = string.Join(", ", tranportsWithNoEndpoints);
-                throw new InvalidOperationException($"At least one endpoint needs to be configured for transport(s): {endpointNames}");
+                logger.LogWarning($"The following transport(s) have no endpoints: {endpointNames}");
+            }
+
+            if (tranportsWithNoEndpoints.Count() == transportConfigurations.Count)
+            {
+                throw new InvalidOperationException($"No transport has an endpoint configured. At least one transport should have an endpoint to be able to bridge messages");
```

Comment:
No string interpolation used:

```suggestion
                throw new InvalidOperationException("No transport has an endpoint configured. At least one transport should have an endpoint to be able to bridge messages");
```